File: c/1-10 arası örnekler/python/20-30 arası örnekler/22.kutlesi_hizi_yuksekligi_girilen_maddenin_potansiyel_ve_kinetik_enerjisini_hesaplama.py
m=int(input("Maddenin kütlesini giriniz (m) : "))
v=int(input("Maddenin hızını giriniz (v) :"))
h=int(input("Yüksekliği giriniz (h) :"))

yerCekimSabiti = 9.8
ep=m*yerCekimSabiti*h
ek=(m*v**2)/2

print("potansiyel enerji: ",ep)
print("Kinetik enerji: ",ek)

File: c/20-30 arası örnekler/22_kutlesi_hizi_yuksekligi_girilen_maddenin_potansiyel_ve_kinetik_enerjisini_hesaplama.c
#include <stdio.h>
#include <math.h>

int main()
{
    int m, v, h, ep, ek;
    printf("Maddenin kutlesini giriniz: ");
    scanf("%d", &m);
    printf("Maddenin yerden yuksekligini giriniz: ");
    scanf("%d", &v);
    printf("Maddenin hizini giriniz: ");
    scanf("%d", &h);

    ep = m * 9.8 * h;
    ek = (m * pow(v, 2)) / 2;

    printf("Maddenin potansiyel enerjisi: %d\n", ep);
    printf("Maddenin kinetik enerjisi: %d\n", ek);

    return 0;
}